### PR TITLE
Admin panel continue

### PR DESCRIPTION
**Pull request by mikze**

### CODE REVIEW

File: app/imports/components/Users/AdminPanel/AdminPanel.jsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                    <CardTitle
+                        title="Retro Tool - Admin Panel"
+                        subtitle="Admin"
+                    />
+                    <RaisedButton
+                        onTouchTap={this.changeOption}
+                        label="Manage Users"
+                    />
+                    <RaisedButton
+                        onTouchTap={this.changeOption}
```
Comment (Avogardo) [inline app/imports/components/Users/AdminPanel/AdminPanel.jsx]: I can jump between UsersList and ProjectsList using only one of this buttons ;p

File: app/imports/components/Users/AdminPanel/UsersAndProjectsManagement/ProjectsList/singleProjectView/schema.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import SimpleSchema from 'simpl-schema';
+
+export const schema = new SimpleSchema({
+    moderators: {
+        type: Array,
+        defaultValue: 'mikze',
```
Comment (mstanuch) [inline app/imports/components/Users/AdminPanel/UsersAndProjectsManagement/ProjectsList/singleProjectView/schema.js:6]: Nice try ;)

File: app/imports/api/projects/actions.js [hunk 3]
```diff
@@ -18,10 +22,32 @@ const createNewProject = (name, moderators, members) => new Promise((resolve, re
...
+    }, (err, res) => {
+        if (err) {
+            err = err.reason || err;
+            return reject(new Error(err));
+        }
+        return resolve(res);
+    });
+});
+
+const removeMod = (projectId, userId) => {
```
Comment (mstanuch) [inline app/imports/api/projects/actions.js:44]: Use full name `removeModerator`

File: app/imports/components/Users/AdminPanel/UsersAndProjectsManagement/ProjectsList/ProjectsList.jsx [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+        };
+
+        this.openDialog = this.openDialog.bind(this);
+        this.closeDialog = this.closeDialog.bind(this);
+        this.filtr = this.filtr.bind(this);
+        this.addMode = this.addMode.bind(this);
+        this.removeMod = this.removeMod.bind(this);
+    }
+
+    filtr(project) {
```
Comment (mstanuch) [inline app/imports/components/Users/AdminPanel/UsersAndProjectsManagement/ProjectsList/ProjectsList.jsx:29]: Typo

File: app/imports/components/Users/AdminPanel/UsersAndProjectsManagement/ProjectsList/ProjectsList.jsx [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+            filter: '',
+            openDialog: false,
+            project: [],
+            moderators: [],
+        };
+
+        this.openDialog = this.openDialog.bind(this);
+        this.closeDialog = this.closeDialog.bind(this);
+        this.filtr = this.filtr.bind(this);
+        this.addMode = this.addMode.bind(this);
```
Comment (mstanuch) [inline app/imports/components/Users/AdminPanel/UsersAndProjectsManagement/ProjectsList/ProjectsList.jsx:25]: `Mod`, `Mode` - please use full name `Moderator`.

File: app/imports/components/Users/AdminPanel/UsersAndProjectsManagement/ProjectsList/ProjectsList.jsx [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+        this.removeMod = this.removeMod.bind(this);
+    }
+
+    filtr(project) {
+        return project.name.search(this.state.filter) >= 0;
+    }
+
+    openDialog(project) {
+        this.setState({ openDialog: true });
```
Comment (mstanuch) [inline app/imports/components/Users/AdminPanel/UsersAndProjectsManagement/ProjectsList/ProjectsList.jsx:34]:
```
this.setState({ 
    openDialog: true,
    moderators: project.moderators,
    project,
});
```

File: app/imports/components/Users/AdminPanel/UsersAndProjectsManagement/UsersList/UsersList.jsx [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+        super(props);
+
+        this.state = {
+            filter: '',
+        };
+
+        this.filtr = this.filtr.bind(this);
+    }
+
+    filtr(user) {
```
Comment (mstanuch) [inline app/imports/components/Users/AdminPanel/UsersAndProjectsManagement/UsersList/UsersList.jsx:23]: Typo

File: app/imports/components/Users/AdminPanel/UsersAndProjectsManagement/ProjectsList/ProjectsList.jsx [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+
+export default class ProjectsList extends React.Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            filter: '',
+            openDialog: false,
+            project: [],
```
Comment (mstanuch) [inline app/imports/components/Users/AdminPanel/UsersAndProjectsManagement/ProjectsList/ProjectsList.jsx:18]: One or more?
Project or projects?

File: app/imports/components/Users/AdminPanel/UsersAndProjectsManagement/ProjectsList/ProjectsList.jsx [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+            moderators: [],
```
Comment (mstanuch) [inline app/imports/components/Users/AdminPanel/UsersAndProjectsManagement/ProjectsList/ProjectsList.jsx:19]: You can get moderators reactively.

**Review by mstanuch — CHANGES_REQUESTED** [review]
I reviewed a part of your changes. Before I review everything, please merge branch devel into your branch.

USE: git merge
https://git-scm.com/docs/git-merge

**Comment by mstanuch** [comment]
Please reopen this pull request. Select `devel` as base branch.

**Pull request closed without merging**